src/main/groovy/com/morpheusdata/hyperv/HyperVCloudProvider.groovy

```diff
@@ -794,31 +856,31 @@ class HyperVCloudProvider implements CloudProvider {
 				}
 				if (match) {
 					server.uniqueId = match.ID
-					//server.save(flush: true)
 					context.services.computeServer.save(server)
 				}
 			}
 		}
 	}
 
-	def listVirtualMachines(opts) {
+	def listVirtualMachines(opts, testVmConnFlg) {
```

Comment:
I don't think we need this method, just call the API service directly.

Reply:
review comment changes added as suggested